src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java

```diff
@@ -47,6 +50,28 @@ public void blockedUserShouldSeeMessageOnArticleEdit() {
     edit.verifyBlockedUserMessage();
   }
 
+  @Test(groups = {"usersAndRights002", "UsersAndRights"}, dependsOnMethods = {
+      "staffCanBlockUser"})
+  @Execute(asUser = User.BLOCKED_USER)
+  public void blockedUserShouldBeAbleToChangeEmail() {
+      final String newEmailAddress = "myAwesomeEmail@email.co.uk";
```

Comment:
if it is `final` why it is not `static`?
if it is `final static` move this field to class field